core/test/Cardano/Address/ScriptSpec.hs

```diff
@@ -545,7 +545,7 @@ spec = do
                 `shouldBe` Left err
 
         it "Invalid JSON" $ do
-            let err = "Error in $: Failed reading: not a valid json value at ''';[]['"
```

Comment:
This one failed on a fresh clone, but not in the CI apparently :thinking:

Reply:
Seems like it could be due to different aeson versions.
But it's rather odd that `stack` builds get different results on CI compared to locally.

Reply:
Which one should be the source of truth?